packages/conventional-changelog-lmc-github/src/templates/template.hbs

```diff
@@ -0,0 +1,17 @@
+{{> header}}
+
+{{#each commitGroups}}
+
+  {{#if title}}
+### {{title}}
+  {{/if}}
+
```

Comment:
I would personally prefer not to have this extra newline after group header. It looks more clear and natural to me if the commits follow right after the header.

Compare this:
```md
### Bug Fixes
* Tell bundlers not to bundle `crypto` dependency ([22ee445](https://github.com/lmc-eu/cookie-consent-manager/commit/22ee445))

### Code Refactoring
* Move browser example into browser directory ([f6893ee](https://github.com/lmc-eu/cookie-consent-manager/commit/f6893ee))

### Dependencies
* Ignore stylelint updates ([2a25de1](https://github.com/lmc-eu/cookie-consent-manager/commit/2a25de1))
* Pin dependencies ([5c9feb7](https://github.com/lmc-eu/cookie-consent-manager/commit/5c9feb7))
```

With this:
```md
### Bug Fixes

* Tell bundlers not to bundle `crypto` dependency ([22ee445](https://github.com/lmc-eu/cookie-consent-manager/commit/22ee445))

### Code Refactoring

* Move browser example into browser directory ([f6893ee](https://github.com/lmc-eu/cookie-consent-manager/commit/f6893ee))

### Dependencies

* Ignore stylelint updates ([2a25de1](https://github.com/lmc-eu/cookie-consent-manager/commit/2a25de1))
* Pin dependencies ([5c9feb7](https://github.com/lmc-eu/cookie-consent-manager/commit/5c9feb7))
```

Reply:
(also I'm used to this from keep-a-changelog format :shrug: https://keepachangelog.com/en/1.0.0/ )

Reply:
This formatting is from [standard templates](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages) like angular, eslint even conventional commits. They all have this extra newline. I also think that it is good practice to have a blank line after header. 

But this can be solved by s[ome rules](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md022---headers-should-be-surrounded-by-blank-lines) :D